config/initializers/99_clowder-config.rb

```diff
@@ -0,0 +1,43 @@
+
+if ClowderCommonRuby::Config.clowder_enabled?
+
+  config = ClowderCommonRuby::Config.load
+
+  # TO-DO: Refactor with a helper function (get_service_url(app_name, component)
+  host_inventory_hostname = config.private_dependency_endpoints.host-inventory.service.hostname
+  host_inventory_port = config.private_dependency_endpoints.host-inventory.service.port
+  host_inventory_url = "#{host_inventory_hostname}:#{host_inventory_port}"
+
+#  Remediations is not yet clowderized, effort tracked as https://issues.redhat.com/browse/RHCLOUD-10212
+#  remediations_hostname = config.private_dependency_endpoints.remediations.service.hostname
+#  remediations_port = config.private_dependency_endpoints.remediations.service.port
+#  remediations_url = "#{remediations_hostname}:#{remediations_port}"
+  remediations_url = "" # TODO: Read from environment
+
+  rbac_hostname = config.private_dependency_endpoints.rbac.rbac.hostname
+  rbac_port = config.private_dependency_endpoints.rbac.rbac.port
+  rbac_url = "#{rbac_hostname}:#{rbac_port}"
+
+  prometheus_exporter_hostname = config.private_dependency_endpoints.compliance.prometheus-exporter.hostname
+  prometheus_exporter_port = config.private_dependency_endpoints.compliance.prometheus-exporter.port
+
+  # TO-DO add in-memory db here for compliance-redis
+
+  Settings.add_source!(Rails.root.join('config', 'clowder.yml'))
```

Comment:
Are you going to add this file as part of this PR?

Reply:
no that was an overlook, I'm removing it. Clowder config is only meant to be read through this config library object, so there's little value on using a YAML file this templating